crnlib/crn_find_files.cpp

```diff
@@ -233,18 +233,19 @@ bool find_files::find_internal(const char* pBasepath, const char* pRelpath, cons
       }
     }
 
+    dynamic_string filename(ep->d_name);
+    dynamic_string fullname = pathname + filename;
+
     if (!known) {
-      console::error("Cannot detect if the given path is a file or a directory");
-      return false;
+      console::warning("Cannot detect if this path is a file or a directory: %s", fullname.get_ptr());
+      continue;
     }
 
     if (!is_file && !is_directory) {
-      console::error("The given path is not a file neither a directory");
-      return false;
+      console::warning("This path is not a file neither a directory: %s", fullname.get_ptr());
+      continue;
     }

```

Comment:
```suggestion
  if (!known || (!is_file && !is_directory)) {
      console::warning("Ignoring unsupported path: %s", fullname.get_ptr());
      continue
  }
```

Reply:
Right, I liked the way we knew there was a failure at detecting file paths, but we should start trusting our code as we now implement a guessing that is standard. A single warning can do it.

I like the “Ignoring…“ wording, it's better, thanks.

Dibe?